boards/arm/cy8ckit_062_wifi_bt_m0/doc/index.rst

```diff
@@ -128,13 +128,9 @@ Cy_WDT_Disable().
 
 Build the project for CM0+
 
-.. code-block:: console
-
-   $ cd samples/hello_world
-   $ mkdir build
-   $ cd build
-   $ cmake -G"Eclipse CDT4 - Ninja" -DBOARD=cy8ckit_062_wifi_bt_m0 ..
```

Comment:
Hmm, I don't think so, this is specifically about using the eclipse generator; the replacement is different, no?

Reply:
I think this was a mistake from the author of the doc, because there is no other reference to Eclipse in the file and I tried building the sample with ninja for that particular board and it worked fine.

Reply:
I checked with one of the authors, @OlegKapshii and he confirmed that this can be `-GNinja`.

Reply:
Saw your slack conversation. Thanks for checking.